### PR TITLE
User log in & log out

### PR DESCRIPTION
**Pull request by hunguyen1702**
---
This change is [Reviewable](https://reviewable.io/reviews/framgia/fels_nvh/9)

### CODE REVIEW

**Comment by hunguyen1702** [comment]
ready

File: app/views/layouts/_header_logged_in.html.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<div class="navbar-fixed">
+  <ul id="dropdown1" class="dropdown-content">
+    <li>
+      <a href="#!" class="a-dropdown">
+        <i class="material-icons dropdown-icons">contacts</i>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header_logged_in.html.erb]: I18n

File: app/views/layouts/_header_logged_in.html.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <%= t "header.profile_setting" %>
+      </a>
+    </li>
+    <li>
+      <a href="#!" class="a-dropdown">
+        <i class="material-icons dropdown-icons">collections_bookmark</i>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header_logged_in.html.erb]: I18n

File: app/views/layouts/_header_logged_in.html.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        </li>
+        <li>
+          <a href="#about">
+            <%= t "header.word_list" %>
+          </a>
+        </li>
+        <li>
+          <a class="dropdown-button" href="#!" data-activates="dropdown1">
+            <%= t "header.account" %>
+            <i class="material-icons right" id="dd-arrow">arrow_drop_down</i>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header_logged_in.html.erb]: I18n

File: app/views/sessions/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            <%= f.label :password, for: :password %>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col s5">
+          </div>
+          <div class="col s2">
+          </div>
+          <div class="col s5">
+            <%= f.check_box :remember_me, class: "rm-checkbox", id: "test5" %>
```
Comment (huongnguyenmta) [inline app/views/sessions/new.html.erb]: tên `id` selector c đặt cho rõ nghĩa nhé

**Comment by huongnguyenmta** [comment]
commented

**Comment by hunguyen1702** [comment]
ready

**Pull request merged**